### DISCUSSION TITLE
Need Help Enabling A Self-developed Sublime Text Plugin

### DISCUSSION THREAD

**Discussion by lemorage** [sublimetext-io/docs.sublimetext.io#93] [Q&A] [answered]
Hi everyone,

I recently developed a small plugin following the instructions outlined in the ST community documentation. I've saved it in the `Packages/my_plugin/` directory, but now I'm a bit stuck on how to actually enable and use it through the command palette.

I've searched for my plugin's commands in the command palette, but I can't find them. Can you guide how to properly enable a self-created plugin in Sublime Text? I didn't see any more information about this issue in the docs.

Any insights would be greatly appreciated. Thanks!

**Comment by keith-hall** [comment, marked as answer]
Hi, plugins are enabled by default. It seems likely that something went wrong loading or executing your code. Are there any errors in the ST console (View menu -> Show Console)?

**Reply by FichteFoll** [reply]
And, after having checked that, did you create a `.sublime-commands` file for your newly created command? Can you run it from the console (using `window.run_command('your_command_name', args_as_a_dict_if_needed)`)?

**Reply by lemorage** [reply]
Ah, version differences. Turns out I used a slightly newer method that Python 3.3 didn't vibe with😅. Honestly, I really didn't notice that at first. Thanks a bunch for the heads up and all the helpful info. Much appreciated!

**Reply by FichteFoll** [reply]
You can tell ST to use the Python 3.8 plugin host for your package by creating a file named `.python-version` in your package folder with the contents `3.8`.

**Reply by lemorage** [reply]
@FichteFoll Good to know. Really useful stuff, cheers!
